.github/workflows/cypress.yml

```diff
@@ -28,6 +28,22 @@ jobs:
           node-version: 16
       - name: Install root dependencies
         run: yarn install
+      - name: Download and unzip latest wallet
+        run: curl -s https://api.github.com/repos/vegaprotocol/vegawallet/releases/latest | awk -F\" '/browser_download_url.*linux-amd64.zip/{print $(NF-1)}' | xargs wget && unzip vegawallet-linux-amd64.zip && rm vegawallet-linux-amd64.zip
+      - name: Initialize wallet
+        run: ./vegawallet init -f
+      - name: Create recovery phrase file
+        run: echo ${{ secrets.TRADING_TEST_VEGA_WALLET_RECOVERY }} > recovery-phrase.txt
+      - name: Create recovery phrase file
+        run: echo "123" > passphrase.txt
```

Comment:
chore (blocking): hard coded passsword should be a secret and also more secure ;)

Reply:
Now using secret for passphrase